test/restart/mzcompose.py

```diff
@@ -63,6 +63,28 @@ def workflow_audit_log(c: Composition) -> None:
         raise Exception("audit logs not equal after restart")
 
 
+def workflow_timelines(c: Composition) -> None:
+    for _ in range(3):
+        c.start_and_wait_for_tcp(
+            services=[
+                "zookeeper",
+                "kafka",
+                "schema-registry",
+                "materialized",
+                "localstack",
```

Comment:
There is no need to start a `localstack` for this test, it is only required for tests that use S3/SQS and other such.

Reply:
Removed